src/cli/commands/force/lightning/lwc/test/utam/spec/test.spec.js

```diff
@@ -0,0 +1,14 @@
+const Eula = require('salesforce-pageobjects/salesforceapp/pageObjects/authentication/eula');
+const Login = require('salesforce-pageobjects/helpers/pageObjects/login');
+
+describe('Login to Salesforce App', () => {
+    it('testLogin', async () => {
+        const eula = await utam.load(Eula);
+        await eula.accept();
+
+        utam.setBridgeAppTitle('Login | Salesforce');
+        const login = await utam.load(Login);
+
+        await login.loginToHomePage('foo@bar.com', 'test1234', '');
+    });
+});
```

Comment:
Sample test spec added so that it is easier to test and debug the command.

Reply:
If we decide to keep the sample here it will be the first time we include a file to input to test a command.